docs/docs/features/encoders.md

```diff
@@ -23,25 +23,37 @@ Keyboards and macropads with encoder support will typically take the two EC11 pi
 
 ### Rotation
 
-Rotation is handled separately as a type of sensor. The behavior for this is set in `sensor-bindings`, which is defined in each keymap layer in the following format:
+Rotation is handled separately as a type of sensor. The behavior for this is set in `sensor-bindings`. See [Sensor Rotation](../behaviors/sensor-rotate.md) for customizing this behavior.
 
-```
-sensor-bindings = <BINDING CW_KEY CCW_KEY>;
-```
-
-- `BINDING`, for now, has only one behavior available; `&inc_dec_kp` for key presses (see [Key Press](../behaviors/key-press.md) for details on available keycodes).
-- `CW_KEY` is the keycode activated by a clockwise turn.
-- `CCW_KEY` is the keycode activated by a counter-clockwise turn.
-
-Additional encoders can be configured by adding more `BINDING CW_KEY CCW_KEY` sets immediately after the first.
+Additional encoders can be configured by adding more bindings immediately after the first.
 
 As an example, a complete `sensor-bindings` for a Kyria with two encoders could look like:
 
 ```
-sensor-bindings = <&inc_dec_kp C_VOL_UP C_VOL_DN &inc_dec_kp PG_UP PG_DN>;
+/ {
+    behaviors {
+        rgb_encoder: rgb_encoder {
+            compatible = "zmk,behavior-sensor-rotate";
+            label = "RGB_ENCODER";
+            #sensor-binding-cells = <0>;
+            bindings = <&rgb_ug RGB_BRD>, <&rgb_ug RGB_BRI>;
+        };
+        arrow_encoder: arrow_encoder {
+            compatible = "zmk,behavior-sensor-rotate";
+            label = "ARROW_ENCODER";
+            #sensor-binding-cells = <0>;
+            bindings = <&kp LEFT>, <&kp RIGHT>;
```

Comment:
Looking at this...  I'm not sure I love how much this complicates the simple case. It might be really nice to have an equivalent generic behavior (this one, but w/ tweaks?) that takes a binding cells value of 2, and bindings *without* params, and forwards on the params bound to it.

It would allow us to redefine `inc_dec_kp` like:

```
/ {
	behaviors {
		/* DEPRECATED: `inc_dec_cp` will be removed in the future */
		/omit-if-no-ref/ inc_dec_cp: inc_dec_kp: behavior_sensor_rotate_key_press {
			compatible = "zmk,behavior-sensor-rotate-var";
			label = "ENC_KEY_PRESS";
			#sensor-binding-cells = <2>;
                        bindings = <&kp>, <&kp>;
		};
	};
};
```

Thoughts?

Reply:
Added this to the pr